packages/mds-attachment-service/@types/index.ts

```diff
@@ -14,9 +14,13 @@ export interface AttachmentDomainModel {
 export type AttachmentDomainCreateModel = DomainModelCreate<AttachmentDomainModel>
 
 export interface AttachmentService {
+  deleteAttachment: (attachment_id: UUID) => AttachmentDomainModel
   name: () => string
+  writeAttachment: (file: Express.Multer.File) => AttachmentDomainModel
 }
 
 export const AttachmentServiceDefinition: RpcServiceDefinition<AttachmentService> = {
-  name: RpcRoute<AttachmentService['name']>()
+  deleteAttachment: RpcRoute<AttachmentService['deleteAttachment']>(),
+  name: RpcRoute<AttachmentService['name']>(),
```

Comment:
Could probably remove `name` from the service now. It was just a placeholder when there were no other methods.